src/datasource.test.ts

```diff
@@ -142,260 +137,6 @@ describe('OpenSearchDatasource', function (this: any) {
     });
   });
 
-  describe('When issuing metric query with interval pattern', () => {
-    let requestOptions: any, parts: any, header: any, query: any, result: any;
-
-    beforeEach(async () => {
-      createDatasource({
-        url: OPENSEARCH_MOCK_URL,
-        jsonData: {
-          database: '[asd-]YYYY.MM.DD',
-          interval: 'Daily',
-          version: '1.0.0',
-        } as OpenSearchOptions,
-      } as DataSourceInstanceSettings<OpenSearchOptions>);
-
-      datasourceRequestMock.mockImplementation((options) => {
-        requestOptions = options;
-        return Promise.resolve({
-          data: {
-            responses: [
-              {
-                aggregations: {
-                  '1': {
-                    buckets: [
-                      {
-                        doc_count: 10,
-                        key: 1000,
-                      },
-                    ],
-                  },
-                },
-              },
-            ],
-          },
-        });
-      });
-
-      query = {
-        range: {
-          from: toUtc([2015, 4, 30, 10]),
-          to: toUtc([2015, 5, 1, 10]),
-        },
-        targets: [
-          {
-            alias: '$varAlias',
-            bucketAggs: [{ type: 'date_histogram', field: '@timestamp', id: '1' }],
-            metrics: [{ type: 'count', id: '1' }],
-            query: 'escape\\:test',
-          },
-        ],
-      };
-
-      result = await ctx.ds.query(query).toPromise();
-
-      parts = requestOptions.data.split('\n');
-      header = JSON.parse(parts[0]);
-    });
-
-    it('should translate index pattern to current day', () => {
-      expect(header.index).toEqual(['asd-2015.05.30', 'asd-2015.05.31', 'asd-2015.06.01']);
-    });
-
-    it('should not resolve the variable in the original alias field in the query', () => {
-      expect(query.targets[0].alias).toEqual('$varAlias');
-    });
-
-    it('should resolve the alias variable for the alias/target in the result', () => {
-      expect(result.data[0].name).toEqual('resolvedVariable');
-    });
-
-    it('should json escape lucene query', () => {
-      const body = JSON.parse(parts[1]);
-      expect(body.query.bool.filter[1].query_string.query).toBe('escape\\:test');
-    });
-  });
-
-  describe('When using sigV4 and POST request', () => {
```

Comment:
Moved these tests inside of the `PPL Queries` test block since only PPL time series format queries are still run on the frontend in the dashboard.